README.md

```diff
@@ -59,10 +59,11 @@ On the command line start a cqlsh shell:
 2. Chose to run the Cassandra automatically during start-up
 
 3. Run the setup cql scripts to create the schema and populate the weather stations table.
-On the command line start a cqlsh shell:
 
+On the command line start a cqlsh shell:
 
-    cd c:/path/to/killrweather c:/pat/to/cassandara/bin/cqlsh
+    cd c:/path/to/killrweather
+    c:/path/to/cassandara/bin/cqlsh
```

Comment:
We don't assume you have apache-cassandra-{version}/bin in your path (I do but others may not). So you cd to the data dir of killrweather then start cqlsh.